workers/ohsome_quality_analyst/geodatabase/client.py

```diff
@@ -195,6 +195,13 @@ async def get_available_regions() -> FeatureCollection:
     return feature_collection
 
 
+async def get_regions():
+    query = "SELECT  name, ogc_fid FROM regions"
+    async with get_connection() as conn:
+        record = await conn.fetch(query)
+    return record
```

Comment:
Here you probably want to not return the list of record object but instead return a list of dict or tuples. 
See also the docs https://magicstack.github.io/asyncpg/current/api/index.html#record-objects

``[tuple(record) for record in record_list]```